### PR TITLE
2022q1: dpaa2: Initial draft of the status report.

### PR DESCRIPTION
**Pull request by dsalychev**

### CODE REVIEW

File: 2022q1/dpaa2.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+LX2160A and has a lack of the on-board GbE support in FreeBSD. These drivers are
+about to change it.
+
+It's still work in progress and a rough implementation of the drivers for some
+DPAA2 objects is available at the moment. A new network interface (dpni) can be
+used on Honeycomb for tests.
+
+TODO:
+
+* Update dpaa2_mc_fdt driver for link:https://www.crowdsupply.com/traverse-technologies/ten64[Traverse Ten64] board
```
Comment (pauamma) [inline 2022q1/dpaa2.adoc:27]: s/for/for the/

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q1/dpaa2.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Some of the NXP SoCs (LX2160A, LS1088A) are shipped with
+link:https://www.nxp.com/design/qoriq-developer-resources/second-generation-data-path-acceleration-architecture-dpaa2:DPAA2[DPAA2],
+the second generation of the data path acceleration architecture. It allows to
+dynamically configure and wire packets processing "objects" (like DPNI for a
+network interface, DPMAC for media access controller, etc.) together to form a
+network-on-a-chip.
+
+link:https://solidrun.atlassian.net/wiki/spaces/developer/pages/197494288/HoneyComb+LX2+ClearFog+CX+LX2+Quick+Start+Guide[Honeycomb LX2],
+as one of the powerful ARM64 boards from SolidRun, is also built around NXP
+LX2160A and has a lack of the on-board GbE support in FreeBSD. These drivers are
```
[the remark this answers is not in the document]
Comment (dsalychev) [reply 2022q1/dpaa2.adoc]: done

File: 2022q1/dpaa2.adoc (hunk 1 above)
Comment (dsalychev) [reply 2022q1/dpaa2.adoc]: done

File: 2022q1/dpaa2.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+* Update dpaa2_mc_fdt driver for link:https://www.crowdsupply.com/traverse-technologies/ten64[Traverse Ten64] board
+
+* Review and fix DPAA2 resources allocation to support as much DPNIs as possible
+
+* Free driver resources properly
+
+* Implement CPU affinity for DPIOs and DPNIs
+
+* Implement cached memory-backed software portals
```
[the remark this answers is not in the document]
Comment (dsalychev) [reply 2022q1/dpaa2.adoc:35]: If they have any DPAA2-capable board and drop me a note asking about those portals, they'll be able to do so on their own. These software portals are necessary to work with Queue and Buffer Manager (QBMan) implemented by NXP. They can be configured in different ways, i.e. to be backed by QBMan memory (which is "cache-inhibited") or just RAM (which is "cache-enabled"). DPAA2 TRM is available for everyone as well.

**Review by pauamma — APPROVED** [review]

**Pull request merged**